wolfcrypt/src/sp_c32.c

```diff
@@ -23191,18 +23191,31 @@ static int sp_256_ecc_mulmod_9(sp_point_256* r, const sp_point_256* g, const sp_
 {
 #ifndef FP_ECC
     return sp_256_ecc_mulmod_win_add_sub_9(r, g, k, map, ct, heap);
+#else
+#if defined(WOLFSSL_SMALL_STACK) && !defined(WOLFSSL_SP_NO_MALLOC)
+    sp_digit* tmp;
 #else
     sp_digit tmp[2 * 9 * 6];
+#endif
     sp_cache_256_t* cache;
     int err = MP_OKAY;
 
+#if defined(WOLFSSL_SMALL_STACK) && !defined(WOLFSSL_SP_NO_MALLOC)
+    tmp = (sp_digit*)XMALLOC(sizeof(sp_digit) * 2 * 9 * 6, heap, DYNAMIC_TYPE_ECC);
```

Comment:
I do not see the corresponding XFREE? Is `DYNAMIC_TYPE_ECC` the right type?

Reply:
fixed